<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>623</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>555</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Библиотечная база данных</string>
  </property>
  <property name="iconSize">
   <size>
    <width>96</width>
    <height>96</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_36">
                <item>
                 <widget class="QLabel" name="label_author">
                  <property name="text">
                   <string>&amp;Автор</string>
                  </property>
                  <property name="buddy">
                   <cstring>authorBook</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="authorBook">
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_37">
                <item>
                 <widget class="QLabel" name="label_name">
                  <property name="text">
                   <string>&amp;Название</string>
                  </property>
                  <property name="buddy">
                   <cstring>nameBook</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="nameBook">
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_38">
                <item>
                 <widget class="QLabel" name="label_number">
                  <property name="text">
                   <string>&amp;Номер</string>
                  </property>
                  <property name="buddy">
                   <cstring>numberBook</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="numberBook">
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_39">
                <item>
                 <widget class="QComboBox" name="subsectionBook">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Художественная литература после 1917</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Художественная литература до 1917</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Зарубежная литература</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Детская литература</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Учебный фонд</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ценный фонд</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_23">
                  <item>
                   <widget class="QCheckBox" name="inStock">
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="text">
                     <string>&amp;Отображать, если книги нет в наличии  </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_40">
                <item>
                 <widget class="QPushButton" name="addBook">
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                  <property name="text">
                   <string>&amp;Добавить</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="searchBook">
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                  <property name="text">
                   <string>&amp;Поиск</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_41">
            <item>
             <widget class="QPushButton" name="bookReaders">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">Margin-raight: 3%;</string>
              </property>
              <property name="text">
               <string>Читатели</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteBook">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">Margin-raight: 3%;</string>
              </property>
              <property name="text">
               <string>&amp;Удалить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="giveBook">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">Margin-raight: 3%;</string>
              </property>
              <property name="text">
               <string>&amp;Выдать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="takeBook">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">Margin-raight: 3%;</string>
              </property>
              <property name="text">
               <string>&amp;Забрать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableBook">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">Margin-raight: 3%;</string>
          </property>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>5</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>6</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>7</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>9</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Автор</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Номер</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>В наличии</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Количество</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Отдел</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Издательство</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_42">
                <item>
                 <widget class="QLabel" name="label_Surname">
                  <property name="text">
                   <string>&amp;Фамилия</string>
                  </property>
                  <property name="buddy">
                   <cstring>surnameStudent</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="surnameStudent">
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_43">
                <item>
                 <widget class="QLabel" name="label_Name">
                  <property name="text">
                   <string>&amp;Имя</string>
                  </property>
                  <property name="buddy">
                   <cstring>nameStudent</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="nameStudent">
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_44">
                <item>
                 <widget class="QLabel" name="label_Class">
                  <property name="text">
                   <string>&amp;Класс</string>
                  </property>
                  <property name="buddy">
                   <cstring>courseStudent</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="courseStudent">
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_46">
                <item>
                 <widget class="QPushButton" name="addStudent">
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                  <property name="text">
                   <string>&amp;Добавить</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="searchStudent">
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">Margin-raight: 3%;</string>
                  </property>
                  <property name="text">
                   <string>&amp;Поиск</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_47">
            <item>
             <widget class="QPushButton" name="studentBooks">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">Margin-raight: 3%;</string>
              </property>
              <property name="text">
               <string>Список книг</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteStudent">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">Margin-raight: 3%;</string>
              </property>
              <property name="text">
               <string>&amp;Удалить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="update">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">Margin-raight: 3%;</string>
              </property>
              <property name="text">
               <string>&amp;Обновить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clear">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">Margin-raight: 3%;</string>
              </property>
              <property name="text">
               <string>&amp;Очистить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableStudent">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">Margin-raight: 3%;</string>
          </property>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>5</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>6</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>7</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>9</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Класс</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кол-во книг</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Год рождения</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Моб. телефон</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Адрес</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
